README.md

```diff
@@ -1,10 +1,39 @@
-# liphdbpp
```

Comment:
I would even add the tango-controls github badge on the README (See https://github.com/tango-controls/TangoTickets/issues/6 for the code to insert)

Reply:
Good idea!